Use_Cases/Opportunity_List/README.md

```diff
@@ -0,0 +1,28 @@
+# Opportunity List
+
+Easily view opportunities in a single screen both in chart and table format
+
+<img  src="opportunity_list.png"  width="300"></img>
```

Comment:
```suggestion
<img  src="opportunity_list.png"></img>
```

Hmm, Github rendering is being particular about this image. From what I've read, removing the width attribute can help, and I'm not sure we need this width to be so narrow. Could we remove it?

Reply:
Sure

Reply:
Actually, I made that change and now the image looks real big. It seems all of the other use case images have a set width of 300. The image looked alright on my end - did it not show on yours or did it have some other issue?

Reply:
Yeah, I'm not seeing it on the PR review screen (even after removing that attribute), but that could just be because it's not merged yet. If you want to put that width back, we can double check after it's merged.